.github/workflows/build_and_test.yml

```diff
@@ -217,6 +217,9 @@ jobs:
       run: |
         python3.6 -m pip install numpy 'pyarrow<3.0.0' pandas scipy xmlrunner plotly>=4.8
         python3.6 -m pip list
+    - name: List Python packages (Python 3.9)
+      run: |
+        python3.9 -m pip list
```

Comment:
oh, interesting .. https://github.com/xinrong-databricks/spark/runs/2724401328?check_suite_focus=true

```
/usr/bin/python3.9: No module named pip
```

it complains that `python3.9` doesn't have `pip` .. but assuming the skip tests checks (https://github.com/apache/spark/runs/2707956326 for https://github.com/apache/spark/commit/c225196be0d18975a3d290b0b9d7283d764be322), seems all packages are installed properly.

Reply:
@dongjoon-hyun did you use conda to install these packages in the docker image?

Reply:
reading the update for docker image (https://github.com/dongjoon-hyun/ApacheSparkGitHubActionImage/commit/e6e1d1a62e1db1b3db878a16002a4704f2c65535) seems like it should have `pip` ... weird ..

Reply:
I see what you mean. Would you think we can afford to install pip again? It takes ~5s including listing the packages.

Reply:
CC @dongjoon-hyun Do you have any insights?

Reply:
Thank you for pinging me, @xinrong-databricks . Let me check for you.

Reply:
I confirmed the issue. We are having 3 python versions and installed in the following sequence.
- Python 3.8
- Python 3.9
- PyPy 3.7

The last thing do and it seems to break something.
```
curl -sS https://bootstrap.pypa.io/get-pip.py | pypy3
```

Let me update the image once more. And, I'll add `plotly` to Python 3.8, too.

Reply:
Thanks @dongjoon-hyun for help! I will put the PR on hold then.